src/feeds.js

```diff
@@ -366,6 +370,60 @@ feeds.fetchEventsFromCalendar_ = function(feed, callback) {
 };
 
 
+/**
+ * Updates the notification alarms
+ */
+feeds.updateNotification = function() {
+  if (!options.get(options.Options.NOTIFICATION_SHOW)) {
+    return;
+  }
+  // If event deleted, then delete alarm
+  if(feeds.nextEvents.length === 0) {
```

Comment:
Add space after `if`. In general, please follow the [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html) as much as possible.